tests/test_socket_server.py

```diff
@@ -85,15 +81,22 @@ async def write(self, **data):
             The data to write.
         """
         st = json.dumps({**data})
-        self.writer.write(st.encode() + b"\r\n")
-        self.log.debug(st)
+        self.writer.write(st.encode() + tcpip.TERMINATOR)
```

Comment:
In the doc string for this function note that the data is not appended with more than just a newline char. Please fix the doc string.

Reply:
Done